src/test/java/hw2/SecondExerciseTest.java

```diff
@@ -0,0 +1,80 @@
+package hw2;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class SecondExerciseTest extends TestBase{
+
+    private static final String EXPECTED_DIFFERENT_ELEMENTS_PAGE_TITLE = "Different Elements";
+    private static final String WIND_CHECKBOX = "Wind";
```

Comment:
Do not create constants for locators, use in directly in method calls, remove all of them